docs/03-tutorials/00-serverless/svls-03-expose-function.md

```diff
@@ -159,7 +160,7 @@ Follow these steps:
 
     - **Subdomain** to determine the host on which you want to expose your Function.
 
-5. In the **Access strategies** section, select the `noop` handler from the drop-down list and leave the default settings with the `GET`, `POST`, `PUT`, `PATCH`, `DELETE`, and `HEAD` methods.
+5. In the **Rules** section, select the `noop` handler from the drop-down list and mark all the methods i.e. `GET`, `POST`, `PUT`, `PATCH`, `DELETE`, and `HEAD`.
 
 6. Select **Create** to confirm changes. The dialog box with the form will close.
```

Comment:
The API Rule for the existing Function. Maybe we do not need any additional sentence since in the next step we check whether we can access our Function through the created HTTPS link?